src/Cache/Engine/ApcuEngine.php

```diff
@@ -181,9 +202,9 @@ public function add($key, $value)
         if ($duration) {
             $expires = time() + $duration;
         }
-        apcu_add($key . '_expires', $expires, $duration);
 
-        return apcu_add($key, $value, $duration);
+        return apcu_add($key, $value, $duration) === true
```

Comment:
Why not use array syntax here too as you have done for `apc_store()`?

Reply:
* Slipped through that _add is actually a write
* Didn't know it had support, for that, too (only discovered myself)
* Will fix

Reply:
After digging a bit into this I think we should not raise warnings for failings of `apc_add`.
`apc_add` is meant to add-or-ignore. If it cannot because the key already exists, the array signature form of `apcu_add` will return those keys it could not insert because they already exist as failures.

That is probably not something we want to have appear in logs anywhere, right?